tests/acceptance/logout-test.js

```diff
@@ -0,0 +1,28 @@
+import Ember from "ember";
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import { authenticateSession } from 'code-corps-ember/tests/helpers/ember-simple-auth';
```

Comment:
Line is too long.<br>'import' is only available in ES6 (use esnext option).<br>Mixed double and single quotes.